### DISCUSSION TITLE
應用「AWS」雲端工具 協助企業數位轉型

### DISCUSSION THREAD

**Discussion by oli-dontsleep** [1101-GS4719-C/Python-Community-GS4719-C#610] [科技新知]
[https://www.thenewslens.com/article/166639](url)
疫情加速數位化時代到來，現在各企業正努力尋求數位的轉型，而AWS提供各項工具，協助工程師利用各種程式語言，開發所需的內容。試問大家之後會想學習使用AWS嗎?

**Comment by d26488399** [comment]
透過AWS，不用很強的電腦就可以用到很強大的AI工具

**Comment by gnsJhenJie** [comment]
AWS真的好用, 不過用雲端服務都要記得算一下價錢, 像我上個月忘記關某個資源就噴掉800多元...

**Comment by Rosancl** [comment]
看到標題提到AWS點進來，但現在要看已經error 404了嗚嗚

**Comment by HsuChen88** [comment]
有使用過幾次的經驗 他是以量計費的 用多少算多少 所以小型部署也可以用

**Comment by leon10456** [comment]
感謝樓主的分享 AWS真的好用 但是有點貴..
